cfep-21.md

```diff
@@ -0,0 +1,60 @@
+
+<table>
+<tr><td> Title </td><td> Conda Forge-Bioconda Transfer Policy </td>
+<tr><td> Status </td><td> Draft </td></tr>
+<tr><td> Author(s) </td><td> Mervin Fansler &lt;mervin.fansler@bric.ku.dk&gt;</td></tr>
+<tr><td> Created </td><td> 10 July 2024</td></tr>
+<tr><td> Updated </td><td> 18 July 2024</td></tr>
+<tr><td> Discussion </td><td> https://github.com/conda-forge/cfep/pull/52 </td></tr>
+<tr><td> Implementation </td><td> NA </td></tr>
+</table>
+
+## Abstract
+
+Bioconda (BC) and Conda Forge (CF) have occasionally transferred package recipes between them. The `strict` channel
+priority recommended to BC users and used when building packages on the BC channel poses an
+issue when transfers occur, especially in the CF-to-BC direction. This proposal aims to 
+define a policy for standard procedure when such transfers occur. Included in this is the introduction
+of a new label to be applied to all package builds that transfer CF-to-BC.
+
+## Specification
+
+### Bioconda to Conda Forge (BC-to-CF) Transfers
+
+1. Packages on BC considered for transfer should be marked with a ["Move to Conda Forge" label](https://github.com/bioconda/bioconda-recipes/labels/Move%20to%20Conda-Forge) in the `bioconda-recipes` repository. This label can be applied either in an Issue or a Pull Request.
+2. A Pull Request should be create on CF's `staged-recipes` repository to add the
+3. The linter will flag when adding a recipe already hosted by BC. This will trigger a message to [the **conda-forge/bioconda-recipes** team](https://github.com/orgs/conda-forge/teams/bioconda-recipes).
+4. Upon approval by a **conda-forge/bioconda-recipes** team member, the Pull Request can then be reviewed and merged.
+5. A Pull Request should be created (or edited) on `bioconda-recipes` that removes the recipe. A link to the CF `staged-recipes` Pull Request should be documented in the Pull Request text.
+
+### Conda Forge to Bioconda (CF-to-BC) Transfers
+
+1. An Issue should be created on the CF feedstock with the title "Move to Bioconda". The **conda-forge/bioconda-recipes** team can also be pinged for their input.
+2. A Pull Request should be created on the `conda-forge/admin-requests` feedstock to label all previous builds of this feedstock with a `transferred-to-bioconda` label. It should also initiate a request from the Core team to archive the feedstock. The **conda-forge/bioconda-recipes** should be pinged to review.
```

Comment:
We might need to add code for this.

But I do see one issue: the package will disappear from main for those that do not use bioconda. What's the recommended practice here? Them adding the label after experience breakage? Is there a way to avoid the breakage from happening?

Reply:
Yes, users would experience an inability to find the package and would have to add the label to use it again. I don't think there is a way around this. It would be discoverable on Anaconda Cloud web searching, but perhaps something should be added to the archived feedstock itself. Can a message be added to the README? or create a pinned Issue with a link to documentation?

Maybe I can put some numbers together on what packages actually have moved before, but in my experience they are already bioinformatics oriented, so my expectation is that most users of such packages will already have Bioconda configured.

Reply:
I mention in the Implementation section that we would add a section under "User Documentation > FAQ" to cover how to use the label.